### DISCUSSION TITLE
How to make only the chapter title font bigger?

### DISCUSSION THREAD

**Discussion by ghost** [koreader/koreader#11635] [Q&A]
I want to disable the built-in book font size in the text style settings, but when I do this, the chapter title also becomes the same size as the text of the book — small. I want to remove the inline font size, but I also want the chapter title to be in a larger font than the book text itself.

What should I click? Please help
Device: PocketBook 633

**Comment by poire-z** [comment]
Use `Style tweaks > Text > Font size and families > [x] Reset main text font size`:
![image](https://github.com/koreader/koreader/assets/24273478/be467767-3632-400d-a647-df1ad69d9a6c)

But if the book is crap and use `<p>` instead of `<h1>`, `<h2>`..., for titles, this won't work, and you'd have to rely to some custom per-book Book style tweak.

**Reply by ghost** [reply]
> Use `Style tweaks > Text > Font size and families > [x] Reset main text font size`:
> ![image](https://github.com/koreader/koreader/assets/24273478/be467767-3632-400d-a647-df1ad69d9a6c)
> 
> But if the book is crap and use `<p>` instead of `<h1>`, `<h2>`..., for titles, this won't work, and you'd have to rely to some custom per-book Book style tweak.
> 

I clicked this button, but the chapter title became much smaller than before. So it means that it uses p instead of h? So sad
Is there maybe another way?

**Reply by poire-z** [reply]
No other easy way.
I guess you can just increase your main font size for this book so the main text looks like what you expect.
